lib/components/dropdown.vue

```diff
@@ -76,6 +67,11 @@
                 type: String,
                 default: null
             }
+        },
+        methods:{
+            log() {
```

Comment:
Should this maybe moved to a `utils` helper?  We do have a `warn` method in utils